etna/packages/etna-js/spec/helpers.js

```diff
@@ -11,15 +11,34 @@ export const stubUrl = ({
   request,
   status = 200,
   headers = {},
-  host = 'http://www.fake.com'
+  host = 'http://localhost'
 }) => {
-  nock(host)
-    [verb](path, request)
-    .reply(status, response, {
-      'Access-Control-Allow-Origin': '*',
-      'Content-type': 'application/json',
-      ...headers
-    });
+  let nocked;
+
+  return new Promise((resolve, reject) => {
+    nocked = nock(host)
+      [verb](path, request)
+      .reply(status, () => {
+        console.log('stubUrl: Received request matching', { verb, path, request });
+        resolve(nocked);
+        return response;
+      }, {
+        'Access-Control-Allow-Origin': '*',
+        'Content-type': 'application/json',
+        ...headers
+      });
+  });
 };
 
 export const cleanStubs = () => nock.cleanAll();
+
+global.expect.extend({
```

Comment:
Useful extension to expect that allows own to ensure that a given promise does resolved.  Used in combination with the above to assert that a given mocked response is actually fulfilled.